### ISSUE TITLE
Keep last status for null/empty data points

### ISSUE THREAD

**Issue by aditya-pande-continuum** [flant/grafana-statusmap#154]
In our monitoring system, we have used PostgreSQL as the backend database to store status data.
The values in the DB(numeric data) are mapped in the following way in the Grafana dashboard:

1 in DB --> Green color --> Success
2 in DB --> Yellow color --> Degraded
3 in DB --> Orange color --> Timeout
4 in DB --> Red color --> Failure
No data point --> Showed null as 0 --> Some other color

Attaching screenshots for reference:

![image](https://user-images.githubusercontent.com/41953314/101356761-933b2280-38be-11eb-8762-748efbab7858.png)
![image](https://user-images.githubusercontent.com/41953314/101356760-933b2280-38be-11eb-8fb8-58fafac466a9.png)

In case of empty/null data points, we would like to have an option to replace them with the status of the last data point.
For e.g., if the latest data point before the null value shows red (failure), the subsequent blank data points should get replaced with status = failure (red), until there is an actual data point.

**Comment by diafour** [comment]
Hello! It seems that this feature is highly demanded! Recent issue #151.

**Comment by continuum-harsh-shete** [comment]
@diafour Any Further update on this?

**Comment by diafour** [comment]
Yes, it is a new feature. No plans for it though, definitely not this year.
